dev/plugins/hu.elte.txtuml.export.plantuml/src/hu/elte/txtuml/export/plantuml/PlantUmlExporter.java

```diff
@@ -61,35 +58,35 @@
 	protected boolean hadErrors = false;
 	protected String errorMessage = null;
 
-	public PlantUmlExporter(String txtUMLProjectName, String generatedFolderName, List<String> SeqDiagramNames) {
+	public PlantUmlExporter(final String txtUMLProjectName, final String generatedFolderName,
+			final List<String> seqDiagramNames) {
```

Comment:
I replaced that List<String> with List<IType> in #589. What do you think about it? Anyway I think we shouldn't change it in this pull request, because replacing the names with something else is a part of fixing a bug in #589, so it will be changed in that pull request.